Library/Formula/gcviewer.rb

```diff
@@ -0,0 +1,27 @@
+class Gcviewer < Formula
+  homepage "https://github.com/chewiebug/GCViewer"
+  url "https://github.com/chewiebug/GCViewer/zipball/master"
+  version "1.34.1"
+  sha256 "775430498a8a38d6bce59e265b948ff47fafb5e00aea3fc5d3c47adcd650a36c"
+
+  depends_on "maven" => :build
+
+  def install
+    libexec.install Dir['*']
+    cd libexec
+
+    system "mvn clean install"
+
+    (bin / "gcviewer.sh").write <<-EOS.undent
+      #!/bin/sh
+
+      java -jar #{libexec}/target/gcviewer-#{version}.jar
+    EOS
+
+    bin.install_symlink "#{bin}/gcviewer.sh" => "gcviewer"
+  end
+
+  test do
+    assert_equal "#{libexec}/target/gcviewer-#{version}.jar", `ls #{libexec}/target/gcviewer-#{version}.jar 2>&1`.strip
```

Comment:
I'm not sure how to test functionality since it's a GUI application and it does not seem to accept command-line arguments (well, it actually does but it assumes that it's a gc log, tries to parse it and returns an exception)...